leader-election-impl/src/main/java/com/palantir/paxos/SqlitePaxosStateLog.java

```diff
@@ -100,5 +109,9 @@ public void truncate(long toDeleteInclusive) {
 
         @SqlUpdate("DELETE FROM <table> WHERE seq <= :seq")
         boolean truncate(@Define("table") String table, @Bind("seq") long seq);
+
+        @SqlBatch("INSERT OR REPLACE INTO <table> (seq, val) VALUES (:round.sequence, :round.valueBytes)")
+        <V extends Persistable & Versionable> boolean[] writeBatchOfRounds(@Define("table") String table,
```

Comment:
Discussed why this must be `boolean[]` offline: required because of the signature of `execute()`